multinet/api.py

```diff
@@ -35,13 +35,25 @@ def get_workspaces() -> Any:
 
 
 @bp.route("/workspaces/<workspace>", methods=["GET"])
+@require_reader
 @swag_from("swagger/workspace.yaml")
 def get_workspace(workspace: str) -> Any:
     """Retrieve a single workspace."""
-    return db.get_workspace(workspace)
+    metadata = db.get_workspace_metadata(workspace)
+    perms = metadata["permissions"]
+
+    return {
+        "name": metadata["name"],
+        "owner": perms["owner"],
+        "maintainers": perms["maintainers"],
+        "writers": perms["writers"],
+        "readers": perms["readers"],
+        "public": perms["public"],
+    }
```

Comment:
Mainly because that includes ArangoDB metadata. So while I was plucking the useful data out of that, I thought I would flatten the structure of the return value as well. This is in the interest of keeping API return values as simple as possible.

Reply:
I recognize your concern, however `metadata["permissions"]` shouldn't contain any arango metadata, since that's included on the root level of `metadata`, and not the subfield of `permissions`.

Reply:
That is a good point. I am also storing the name at top level of the return value. But thinking about it some more, I can see how we might wish to extend this in the future by adding other sections, like a list of tables, list of networks, etc., which could be optionally included in the response. I will do something closer to what you've suggested.

Reply:
See b8185a72d1e30db359f6f2baa4d9cc69ffd3a41c.